### PR TITLE
Sdkauto

### PR DESCRIPTION
**Pull request by leni-msft**

### CODE REVIEW

**Comment by leni-msft** [comment]
@anthony-c-martin @apclouds Here's PR to fully automate schemas generations. It's achieved by auto updating/overriding readme.azureresourceschema.md in the generation pipeline, before invoking autorest. For exceptional case (like keyvault), we can override apiVersion files in [autogenlist.ts](https://github.com/Azure/azure-resource-manager-schemas/pull/1582/files#diff-2c9ca60f6f798ef4a7e07de10e98f779cd7472bd040e06c627d366092636581dR674)
After the PR is merged, I will delete all readme.azureresourceschema.md files in spec repo: https://github.com/Azure/azure-rest-api-specs/pull/13885
Please help check.

**Comment by anthony-c-martin** [comment]
@leni-msft ack! I wanted to discuss this plan with our team - will get back to you shortly.

**Review by anthony-c-martin — APPROVED** [review]
Looks good!

File: schemas/2018-04-01/Microsoft.Databricks.json [hunk 1]
```diff
@@ -429,6 +429,39 @@
           ],
           "description": "The object that contains details of encryption used on the workspace."
         },
+        "loadBalancerBackendPoolName": {
```
Comment (anthony-c-martin) [inline schemas/2018-04-01/Microsoft.Databricks.json:432]: Would you mind reverting all the auto-generated files in this PR? They should be regenerated automatically once it's been merged.

Comment (leni-msft) [reply schemas/2018-04-01/Microsoft.Databricks.json]: Sure, makes sense. Reverted.

**Pull request merged**